Testing/functional/tests/cypress/integration/e2e/dependent/report.js

```diff
@@ -0,0 +1,446 @@
+                    enabled: true,
+                },
+                {
+                    name: "note",
+                    enabled: true,
+                },
+                {
+                    name: "specialAuthorityRequest",
+                    enabled: true,
+                },
+            ],
+        });
+        cy.login(
+            Cypress.env("keycloak.username"),
+            Cypress.env("keycloak.password"),
+            AuthMethod.KeyCloak,
+            "/dependents"
+        );
+    });
+
+    it("Validate Service Selection", () => {
+        const hdid = existingDependent.hdid;
+
+        const cardSelector = getCardSelector(hdid);
+        const tabButtonSelector = getTabButtonSelector(hdid, "report");
+        const tabSelector = `${cardSelector} [data-testid=report-tab]`;
+
+        cy.get(tabButtonSelector)
+            .should("be.visible")
+            .should("not.be.disabled")
+            .should("not.have.class", "disabled")
+            .click();
+
+        cy.get(`${tabSelector} [data-testid=exportRecordBtn] button`).should(
+            "be.disabled",
+            "be.visible"
+        );
+
+        cy.get(`${tabSelector} [data-testid=infoText]`).should(
+            "have.text",
+            " Select a record type above to create a report "
+        );
+
+        // display visual when no record type selected (mobile and desktop)
+        cy.get(`${tabSelector} [data-testid=infoImage]`).should("be.visible");
+        cy.viewport("iphone-6");
+        cy.get(`${tabSelector} [data-testid=infoImage]`).should("be.visible");
+        cy.viewport(1440, 600);
+
+        cy.get(`${tabSelector} [data-testid=reportType]`).select(
+            "COVID‑19 Tests"
+        );
+
+        cy.get(`${tabSelector} [data-testid=exportRecordBtn] button`).should(
+            "be.enabled",
+            "be.visible"
+        );
+
+        cy.get(`${tabSelector} [data-testid=reportType]`).select("");
+
+        cy.get(`${tabSelector} [data-testid=exportRecordBtn] button`).should(
+            "be.disabled",
+            "be.visible"
+        );
+    });
+
+    it.skip("Validate Medication Report", () => {
+        const hdid = existingDependent.hdid;
+
+        const cardSelector = getCardSelector(hdid);
+        const tabButtonSelector = getTabButtonSelector(hdid, "report");
+        const tabSelector = `${cardSelector} [data-testid=report-tab]`;
+
+        cy.get(tabButtonSelector)
+            .should("be.visible")
+            .should("not.be.disabled")
+            .should("not.have.class", "disabled")
+            .click();
+
+        cy.get(`${tabSelector} [data-testid=reportType]`).select("Medications");
+        cy.get(
+            `${tabSelector} [data-testid=medication-history-report-table]`
+        ).should("not.exist");
+
+        cy.viewport("iphone-6");
+        cy.get(
+            `${tabSelector} [data-testid=medication-history-report-table]`
+        ).should("not.exist");
+        cy.viewport(1440, 600);
```

Comment:
We can probably skip the check for the preview on mobile, since if the preview isn't there on desktop, it's not likely to suddenly appear on mobile.